.jenkinsci/cancel-nightly-except-develop.groovy

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env groovy
```

Comment:
How much time does it take to go through existing branches and cancel all of them but develop?

Reply:
could not estimate it, because it worked only for one branch by the moment